study.md

```diff
@@ -28,22 +28,31 @@ Asynchronous - operations that may start in a certain sequence and may end in an
 
 In your own words, describe the event loop and how it works.
 
-```md
-<!-- your answer here -->
-```
+`md
+"The Event Loop is a queue of callback functions."" To me, this means that javascript executes a function, and it if is not being used, it is placed into a queu and waits/listens for an action to execute.
+
+http://stackoverflow.com/questions/21607692/understanding-the-event-loop
+ ```
```

Comment:
So the queue is where the callback functions will wait to be called. The event loop waits for the stack to be empty, then it moves the next thing from the queue to the stack.